wasm/util.go

```diff
@@ -50,3 +54,61 @@ func (u Util) Logger(ctx context.Context) *zerolog.Logger {
 	l := u.logger.With().Ctx(ctx).Logger()
 	return &l
 }
+
+func CreateSchema(req *processorv1.CreateSchemaRequest) (*processorv1.CreateSchemaResponse, error) {
```

Comment:
I would move these two functions into `schema.go`.

Reply:
there's no file called `schema.go`, do you mean add the file under the schema directory? or maybe create a util file under `schema`?

Reply:
I mean to create a file `wasm/schema.go` which contains the code related to the schema wasm host functions.